fe1-web/src/features/evoting/reducer/__tests__/ElectionKeyReducer.test.ts

```diff
@@ -87,20 +92,47 @@ describe('getElectionKeyByElectionId', () => {
       getElectionKeyByElectionId(mockElectionId.valueOf(), {
         [ELECTION_KEY_REDUCER_PATH]: {
           byElectionId: {
-            [mockElectionId.valueOf()]: 'someElectionKey',
+            [mockElectionId.valueOf()]: mockElectionKeyString,
             someOtherId: 'someOtherElectionKey',
           },
         } as ElectionKeyReducerState,
-      }),
-    ).toBeJsonEqual('someElectionKey');
+      })?.equals(mockElectionKey),
+    ).toBeTrue();
```

Comment:
Yes this seems to work, thank you! I did not even try it because I thought `toEqual()` would perform a custom check rather than calling `equals` on the operands 😅